cupy_backends/cuda/libs/cusparselt.pyx

```diff
@@ -106,6 +106,9 @@ cdef extern from '../../cupy_cusparselt.h' nogil:
         const cusparseLtHandle_t* handle, const cusparseLtMatmulPlan_t* plan,
         const void* d_dense, void* d_compressed, driver.Stream stream)
 
+    # Version
+    int cusparseLtGetVersion(int* version)
```

Comment:
As talked offline, let's expose `int CUSPARSELT_VERSION` directly.

Reply:
Need to add `CUSPARSELT_VERSION` to the stub header.